o-fish-ios/Views/Components/Pictures/PhotoCaptureController.swift

```diff
@@ -27,12 +27,16 @@ class PhotoCaptureController: UIImagePickerController {
 
     static func show(reportID: String,
                      source: UIImagePickerController.SourceType,
+                     fromVessel: Bool = false,
                      photoTaken: ((PhotoCaptureController, String) -> Void)? = nil) {
 
         let picker = PhotoCaptureController()
         picker.setup(source)
         picker.reportID = reportID
         picker.photoTaken = photoTaken
+        if fromVessel {
+            picker.imageSizeThumbnails = 375 // Size for VesselRecordHeaderView presentation
```

Comment:
I think we can globally change this size,  this should not impact thumbnail size too much

@am-MongoDB please confirm

Reply:
As we discussed, need to download picture from url, so it's inappropriate chunk of code.